src/schema.ts

```diff
@@ -30,6 +30,7 @@ type Resource {
   path: String
   article: Article
   resourceTypes: [ResourceType]
+  meta: ArticleSubset
```

Comment:
Need to expand ArticleSubset with metaImage when https://github.com/NDLANO/Issues/issues/1052 is ready.